pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -114,10 +115,11 @@ func (a sandboxByCreated) Len() int           { return len(a) }
 func (a sandboxByCreated) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 func (a sandboxByCreated) Less(i, j int) bool { return a[i].createTime.After(a[j].createTime) }
 
-// enforceMaxContainersPerEvictUnit enforces MaxPerPodContainer for each evictUnit.
-func (cgc *containerGC) enforceMaxContainersPerEvictUnit(ctx context.Context, evictUnits containersByEvictUnit, MaxContainers int) {
+// retainLatestContainerPerEvictUnit ensures only the most recent container is kept per evictUnit,
+// removing all older containers.
+func (cgc *containerGC) retainLatestContainerPerEvictUnit(ctx context.Context, evictUnits containersByEvictUnit) {
```

Comment:
Hi  @zhifei92, Curious to know about the `retainLatestContainerPerEvictUnit()` method, Does it retain the latest dead container per pod (currently follows MaxPerPodContainer) and remove all the other dead containers from the pod? Am I right?

Reply:
Thanks @zhifei92 for the information.